### DISCUSSION TITLE
Better LaTeX support

### DISCUSSION THREAD

**Discussion by ievlevpn** [alangrainger/share-note#128] [Vote for upcoming features]
It seems that the current LaTeX support is quite limited. For example `\mathbf{x} + \mathbb{S}^{d-1} - \boldsymbol{\alpha}` exports as follows: https://share.note.sx/todowgkf#R8VjDJc/x+2Lktb3qVpYwyjIG8aTlZ9OEE/TNIvCGwA

So far I've noticed that bold math, blackboard bold, bold greeks and some symbols like `\#` often disappear from the shared file.

**Comment by alangrainger** [comment]
Did you follow the instructions on the [LaTeX Troubleshooting page](https://docs.note.sx/Troubleshooting#mathjax--latex)?

This is what I get: https://share.note.sx/l0cr79ta#Ud8blK7M7xpc2dBlnnvjX52pGDgfvvMMNeycmpObeBA

![image](https://github.com/user-attachments/assets/042e68b6-2bd5-4d54-9fd9-75f1895bcb00)

**Reply by MyeongsooSon** [reply]
I was unable to implement the sample you provided.

As I read through the troubleshooting page, it suggested two methods:
1. just use command `Force re-upload `
2. make "master" obsidian note file for Latex glyphs and 'Force re-upload' the "master" file

I created the "master" file (by simply pasting the LaTeX formula mentioned above) and uploaded it using 'Force re-upload,' but it didn’t work for me.
Could you provide more details about the second method?

**Reply by sylv256** [reply]
How did you do this?

**Comment by KeniAlquist** [comment]
Niche problem I have is that some math symbols are not supported. Simple forced re-upload won't fix it (i.e. \bigoplus, \subseteq, \#)
(https://share.note.sx/rcob1rnr#JZzMjKrCyKwngUYCysKYIR0Thj8T5r7Gcifr01MdSHA)

I see the potential of the pluggin for sharing math notes, but this is quite limiting.
